src/main/java/com/example/poc/mapper/CountryNameMapper.java

```diff
@@ -0,0 +1,12 @@
+package com.example.poc.mapper;
+
+import com.example.poc.model.entity.CountryNameEntity;
+import com.example.poc.wsdl_classes.CountryNameResponse;
+
+public class CountryNameMapper {
+    public static CountryNameEntity mapToCountryName(CountryNameResponse countryNameResponse) {
```

Comment:
This can be done in "getCountryName" from "CountryService". You don't need to make a mapper just to map a country name.